web/components/PermissionSelctor.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from "./AccessManagementPage.module.css";
+
+export default function PermissionSelector(props: any) {
```

Comment:
You can define the type for props like this instead of using `any`:
```jsx
interface Name {
    firstName: string;
    lastName: string;
}
function Component({firstName, lastName}: Name){
}
```
Make sure we have type definitions for the props of all components.